---
layout: single
title: To Do List
---


# Stuff going on

My electronic scratchpad of stuff thats happening or needs to happen.
 
- [ ] clean frogs before weekend
- [ ] Check up with Conner and figure out why amps are not working
- [ ] Have Sinthu amp mtDNA in her extractions, if not done already
- [ ] Extract new clivii and froze laevis. Amp DMW in both with positive mtDNA
- [x] **Critique Ben's grant application**
- [ ] presentation for undergrads, mid November
- [ ] presentation for EEB November
- [ ] shell script for mlRho theta
- [x] my own script for pi
- [ ] Maybe finish borealis mapped linkage map
- [ ] Windows of physical distance and map distance (say, 3 markers) across laevis mapped linkage map
- [ ] Summarize results and write method in Duplicate gene paper
- [ ] Add to Duplicate paper newer WGD papers, especially the Lynch 2015 Paramecium paper



# nebulous or less important

- [ ] investigate sex specific reads with X. borealis genome
- [ ] Removing contamination from X. borealis reads?
- [ ] Maybe some Kmer stuff on genome size/low coverage reads with X. borealis HiSeq?
- [ ] with better reference, map sex specific contigs
- [ ] Really focus on making better Borealis reference
- [ ] evolver simulations to see if codeml artificially detects differences more often in older branches than earlier branches.
- [ ] coeff-variation for alpha v beta v time for the difference calculation
- [ ] Summary of alpha v beta v time codeml analyses averaging for mutli-species in one gene
- [ ] trim out out overrepresented sequence in mellotrop single chr seqs seen in FASTqc.
- [ ] SOAPdenovo on mellotrop libraries Caroline cleaned.



---
## Done crap for ref

- [x] Make figures for yn00 analysis for homeolog and blastp comparisons
- [x] Make figure for codeml analysis, but update when new 3 rate model is done
- [x] Evolution conference presentation
- [x] Play with code from Utkarsh
- [x] Gilli/laevis investigate exon26 a bit
- [x] HCG the trop individuals and make families
- [x] SOX3 on the new X. fischbergi individuals (maybe undergrad volunteer can do this)
- [x] Gilli/laevis new Structure and Genetree figures
- [x] Sex linked representatives to Genbank
- [x] Dryad of XML, con trees, sex linked alignments
- [x] Gilli/laevis rewrite paper
- [x] Ensure consistent naming in sex linked alignments
- [x] Investigate duplicate genes, see how often they are weird, like sox3 (could blast in genes from phylo project, as I could know ones that should be in duplicate)
- [x] Sox3 amps in wild samples
- [x] add offspring for sox/fmr/ar if I have them
- [x] mtDNA for gilli/laevis -- clean seq, make tree
- [x] When done, clean new xenopus sequence
- [x] ACE of sex chromosome theories -- some work done, need to investigate feasibility
- [x] rewrite methods section in borealis sex paper
- [x] *X. borealis* Mom, dad and maybe a couple offspring for Sox3 F5 / R1 with R5 internal
- [x] DNAsp for gilli/laevis exon9
- [x] mark midterms
- [x] mark old assignments
- [x] mark latest 4E03 presentation.
- [x] yellow fever vaccination
- [x] remove paraphyletic laevis from exon9 in gilli/lav project
- [x] remove exon25 from paper and figs
- [x] add laevis/gilli labels to tree, also add real gene names
- [x] check that gilli/laevis hybrids are not just paralog amps
- [x] run sox3 borealis 3' amps lower temp, more DNA
- [x] run sox3 3' borealis wild samples on new gel
- [x] run Clivii dmw on higher percent gel
- [x] add genome sequences from X. laevis for all gilli/laevis alignments
- [x] redo structure and gene trees for gilli laevis paper
- [x] find where gilli laevis genes are in laevis genome
- [x] fix per locus gilli introg for exon25
- [x] Silurana instead of xenopus in trees.
- [x] **Summarize all of the analyses done for Dup gene project**, I am starting to forget.
- [x] write pipeable perl script to go to fasta from table (like R script, but want it pipeable).
- [x] come up with a way to calculate the sex specific fraction of a variant site, so that I can map across Xl genome.
- [x] new codeml analyses for duplicate gene stuff. Individual models for alpha and beta with early and late stages.
- [x] when UG on GBS done, map locations of sex specific snps
- [x] Finish draft of Xg--Xl paper, need intro/conclusion broad paragraphs.
- [x] Trimmomatic on mellotrop single chr seqs
- [x] Scythe on mellotrop single chr seqs
- [x] FASTqc on mellotrop single chr seqs
- [x] Figure out overrepresented sequence in mellotrop single chr seqs seen in FASTqc.
- [x] Explore pattern differences in old v new data -- seems to be more b/c patterns.
- [x] Amp mtDNA on other *X. gilli* samples, save ones that worked last time **might need to do a couple more**
- [x] Amp mtDNA on other *X. laevis* East samples,
- [x] DNASP (or new BioPerl script) for *X. gilli* data. **table formatted and in paper**
- [x] Summarize proportion of introgressed individuals for each locus (instead of total)
- [x] Make new figure for summary of introgressed individuals (maybe do new chi2?)
- [x] Read paper for Ben's class tomorrow
- [x] Continue with *X. borealis* genome stuff -- mapping GBS reads? recomb on chr8?
- [x] Variant calls with parents on laevis (expect more heterozygosity in female on chr8)
- [x] Calling pi for mom and dad individually.
- [x] HaplotypeCaller on mom and dad for het sites -- **memory topped out**
- [x] HaplotypeCaller individually on mom and dad for all sites **Most reads removed, I think**
- [x] Why such bad mapping for chr8? Stats did seem funny earlier.
- [x] Summarize what has been done and needs to be done with Continue with *X. borealis* genome stuff
- [x] BlastP of trop seqs to human database
- [x] Maybe mine *X. borealis* HiSeq X data for orthologs of *X. laevis* to reduce missing data in duplicate gene stuff -- **no**
- [x] Use new script to get the blastP matches for the duplicate gene stuff.
- [x] Summarize yn00 analysis of homeolog comparisons
- [x] Summarize yn00 analysis of blastp to humans and dating
- [x] Rerun full analysis for 3 model as tree was wrong
- [x] Make super contig of small  X. laevis matches, match formatting (wrapped fasta). **not needed**
- [x] Redo mapping of *X. boreais* to *X. laeivs* new super contig reference. **not needed**
- [x] finish snp calling with chr7s **most reads thrown out**
